Source/Requests/HASttData.swift

```diff
@@ -0,0 +1,8 @@
+/// Write audio data to websocket, sttBinaryHandlerId is provided by run-start in Assist pipeline
+public struct HASttData: Hashable {
+    var sttBinaryHandlerId: UInt8
+
+    public init(sttBinaryHandlerId: UInt8) {
+        self.sttBinaryHandlerId = sttBinaryHandlerId
+    }
+}
```

Comment:
```suggestion
public struct HASttData: RawRepresentable, Hashable {
    public var rawValue: UInt8

    public init(rawValue: UInt8) {
        self.rawValue = rawValue
    }
}
```

and you can do

```swift
public extension HASttData {
  // idk what this one is called in the docs, just e.g. here
  static var stream: Self = .init(rawValue: 1)
}

public extension HATypedRequest {
    /// Send binary stream STT data
    /// - Returns: A typed request that can be sent via `HAConnection`
    static func sendSttStreamData(data: [UInt8]) -> HATypedRequest<HAResponseVoid> {
        .init(request: .init(type: .sttData(.stream), data: [
            "audioData": data
        ]))
    }
}
```

and then you can use it like the normal, other methods in HAKit.

Reply:
The handler Id will always be dynamic, so it can't be placed on a static var and used in such a way. I renamed `HASttData` to `HASttHandlerId` and created a typed request for it.